src/excalidraw-app/collab/Collab.tsx

```diff
@@ -257,13 +257,6 @@ class Collab extends PureComponent<Props, CollabState> {
       ),
     );
 
-    if (this.portal.socket && this.fallbackInitializationHandler) {
-      this.portal.socket.off(
-        "connect_error",
-        this.fallbackInitializationHandler,
-      );
-    }
-
```

Comment:
unnecessary as there shouldn't be a race condition even if the user manages to stop before initialization timer is done, and is being prompted for confirmation. If the dialog is confirmed after wall clock timer elapses, the callback should still be called in a tick after the sync code after the `window.confirm` is executed.